framework/tester.py

```diff
@@ -272,8 +272,6 @@ def __create_tempesta(self):
         config = ""
         if "config" in desc:
             config = desc["config"]
-        else:
-            raise AttributeError("Tempesta config is empty.")
```

Comment:
I suggest moving this `Exception` to `start_tempesta` [method](https://github.com/tempesta-tech/tempesta-test/blob/ea3ee608d00988c63259cba0ea20392957649bbe/framework/tester.py#L291). 
```python3
    def start_tempesta(self):
        """Start Tempesta and wait until the initialization process finish."""
        # "modules are started" string is only logged in debug builds while
        # "Tempesta FW is ready" is logged at all levels.
        if not self.__tempesta.config.defconfig:
            raise AttributeError("Tempesta config is empty.")
        with dmesg.wait_for_msg("[tempesta fw] Tempesta FW is ready", 1, True):
            self.__tempesta.start()
            if not self.__tempesta.is_running():
                raise Exception("Can not start Tempesta")
```

I think not calling `Exception` is a bad idea for this case.